### ISSUE TITLE
No download button

### ISSUE THREAD

**Issue by merrickw** [bradlys/monochromatic-panda#19]
Hi Bradly - I am seeing some pages where the download button disappears;
Also, I see cases where the download button is left from a previous page and operates on the previous video, not the current video.

Here's an example where no button (or link) appears:
https://www.youtube.com/watch?time_continue=2&v=l5HwNctJJsY

Regards,
Merrick

**Comment by merrickw** [comment]
I found another project/utility here that worked for that video:

https://github.com/rg3/youtube-dl/

**Comment by bradlys** [comment]
Yeah, this is a current limitation with my implementation. Basically, the issue is that YouTube just replaces elements on the page rather than reloads the entire page. My little downloader only works when the whole page is refreshed/changed.

In terms of that video, it seems YouTube is using a different method for their decryption algorithm. Thanks - I'll try to fix it soon.
